src/node.cc

```diff
@@ -947,17 +947,56 @@ void* ArrayBufferAllocator::Allocate(size_t size) {
   return malloc(size);
 }
 
+static bool domainHasErrorHandler(const Environment* env,
+  const Local<Object>& domain) {
```

Comment:
> Oh, and can you s/static/inline/? Using static is not really wrong but inline is arguably more idiomatic C++.

It seems `src/node.cc` is inconsistent regarding the use of `static` and `inline`. It uses either of them, and sometimes both at the same time. Also, what makes `inline` more idiomatic C++?

> EDIT: And uppercase the function name.

Will fix. Why isn't our linter catching that?